microprofile/opentracing-smallrye/src/main/java/org/wildfly/microprofile/opentracing/smallrye/TracerConfigurationConstants.java

```diff
@@ -25,6 +25,8 @@
     public static final String SMALLRYE_OPENTRACING_TRACER_CONFIGURATION = "mp.opentracing.extensions.tracer.configuration";
     public static final String SMALLRYE_OPENTRACING_TRACER_MANAGED = "mp.opentracing.extensions.tracer.managed";
 
+    public static final String MICROPROFILE_CONFIG_CONFIGURATION= "smallrye.opentracing.mp.config";
```

Comment:
This can be any string, yes? IOW we are not exposing some smallrye feature here.

If it can be anything perhaps "wildfly.smallrye.opentracing.mp.config" to further reduce the chances of conflicting with something we don't know about?